presto-ml/src/main/java/com/facebook/presto/ml/MLFeaturesFunctions.java

```diff
@@ -23,94 +23,189 @@
 import com.facebook.presto.spi.type.DoubleType;
 import com.facebook.presto.spi.type.StandardTypes;
 import com.facebook.presto.spi.type.Type;
-import com.google.common.cache.Cache;
-import com.google.common.cache.CacheBuilder;
 import com.google.common.collect.ImmutableList;
-import com.google.common.hash.HashCode;
+
+import java.util.List;
 
 public final class MLFeaturesFunctions
 {
-    private static final Cache<HashCode, Model> MODEL_CACHE = CacheBuilder.newBuilder().maximumSize(5).build();
+    public static final List<Class<?>> ML_FEATURE_FUNCTIONS = ImmutableList.of(Features1.class, Features2.class, Features3.class, Features4.class, Features5.class, Features6.class, Features7.class, Features8.class, Features9.class, Features10.class);
```

Comment:
We should convert the implementation to use varargs. Filed issue: https://github.com/prestodb/presto/issues/10801